typespec/Applications.Core/containers.tsp

```diff
@@ -38,27 +38,27 @@ using OpenAPI;
 namespace Applications.Core;
 
 model ContainerResource is TrackedResource<ContainerProperties> {
-  @doc("Container name")
+  @doc("The name of your resource. See [common values]({{< ref "resource-schema.md#common-values" >}}) for more information.")
```

Comment:
We can't use hugo shortcodes in TypeSpec descriptions because this documentation is used in a lot of other downstream systems where this won't be rendered. I'd keep these descriptions self-contained.

Reply:
additionally, I recommend specifying the specific resource type, i.e. "The name of your **container** resource."